### PR TITLE
Addind subscription count event handler

### PR DESCRIPTION
**Pull request by singhashmeet**
## Description

Added a new event listner for subscription count

## CHANGELOG

* [CHANGED] Describe your change here. Look at CHANGELOG.md to see the format.

### CODE REVIEW

File: PusherClient/Channel.cs [hunk 1]
```diff
@@ -158,5 +177,17 @@ public static ChannelTypes GetChannelType(string channelName)
 
             return channelType;
         }
+
+        public class SubscriptionCountData
+        {
+            [JsonProperty("subscription_count")]
+            public int subscription_count { get; set; }
```
Comment (fbenevides) [inline PusherClient/Channel.cs:184]: Can we keep it as a camelCase attribute? Just to follow the .NET standards

**Comment by amad** [comment]
Let's link this to Jira. Easiest way is to prefix commit message with ticket ID.

File: PusherClient/Channel.cs [hunk 2]
```diff
@@ -20,6 +21,9 @@ public class Channel : EventEmitter
         /// Fired when the Channel has successfully been subscribed to.
         /// </summary>
         internal event SubscriptionEventHandler Subscribed;
+        internal event SubscriptionCountHandler Count;
+
+        public int SubscriberCount = 0;
```
[the remark this answers is not in the document]
Comment (singhashmeet) [reply PusherClient/Channel.cs:26]: Its kind of a convention with this SDK to use short hands
```
pusher.Connected += BindEvents;
pusher.Disconnected += UnbindEvents;
pusher.Subscribed += SubscribedHandler;
pusher.ConnectionStateChanged += StateChangedEventHandler;
pusher.Error += HandleError;
```

**Comment by amad** [comment]
@singhashmeet plz don't forget to update the changelog section of PR description.

**Review by amad — APPROVED** [review]

**Pull request merged**